dockers/docker-macsec/cli-plugin-tests/test_config_macsec.py

```diff
@@ -0,0 +1,146 @@
+import sys
```

Comment:
are these tests running in the build process?

Reply:
Yes, they are.